custom_components/midea_ac/climate.py

```diff
@@ -289,6 +299,21 @@ async def async_set_temperature(self, **kwargs) -> None:
         self._device.target_temperature = round(temperature * 2) / 2
         await self._apply()
 
+    @property
+    def current_humidity(self) -> float | None:
+        """Return the current humidity."""
+        return self._device.indoor_humidity

```

Comment:
We should check `supports_humidity` before returning `indoor_humidity`

Reply:
On second thought. If `supports_humidity` is false, the `indoor_humidity` property will have the default value of None.